frontend/src/helpers/index.ts

```diff
@@ -0,0 +1,19 @@
+//helper method to determine whether app is running in electron environment
```

Comment:
Please add a `space` after `//` and a notice about where this code was taken from:
`// This snipet was taken form the XYZ project (ABC licensed): https://github.com/…`